ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestConfig.java

```diff
@@ -153,12 +153,17 @@
       KSQL_CONFIG_PREFIX + "lag.reporting.enable";
   private static final String KSQL_LAG_REPORTING_ENABLE_DOC =
       "Whether lag reporting is enabled or not. It is disabled by default.";
-
   public static final String KSQL_LAG_REPORTING_SEND_INTERVAL_MS_CONFIG =
       KSQL_CONFIG_PREFIX + "lag.reporting.send.interval.ms";
   private static final String KSQL_LAG_REPORTING_SEND_INTERVAL_MS_DOC =
       "Interval at which lag reports are broadcasted to servers.";
 
+  public static final String KSQL_QUERY_STANDBY_ENABLE_CONFIG =
```

Comment:
this could be set per request? is that why we have this here in addition to KsqlConfig?

Reply:
No . It got wrongly added with the rebase. Good catch

Reply:
It seems like you could just use KSQL_QUERY_PULL_ENABLE_STANDBY_READS rather than both.